src/app/components/CheckoutDialog.css

```diff
@@ -0,0 +1,17 @@
+.addCateringButton {
+  margin-bottom: 1rem;
+}
+.textAlignRight {
+  text-align: right;
+}
+.marginTop1Rem {
+  margin-top: 1rem;
+}
+.lineOfDivision {
```

Comment:
wird sowas auch noch an anderen Stellen benutzt? Generische Sachen sollten wir ja in gemeinsame Daten speichern, und nicht neu in jede einzelne .css

Reply:
die line wird nur einmal benutzt